test/rubygems/test_gem_installer.rb

```diff
@@ -1422,11 +1422,6 @@ def test_find_lib_file_after_install
     write_file File.join(@tempdir, "extconf.rb") do |io|
       io.write <<-RUBY
         require "mkmf"
-
-        CONFIG['CC'] = '$(TOUCH) $@ ||'
```

Comment:
I had to remove these compiler stubs, since it would otherwise try to use dsymutil on the empty files created from `touch`.

Reply:
Rubygems might not assume the compiler, why not remove `CONFIG['dsymutil']` instead?

Reply:
The failures seem coming from `codesign`, and this file won't need changes after 8a89dd2a6cb.

Reply:
> Rubygems might not assume the compiler, why not remove CONFIG['dsymutil'] instead?

Handling CC not referencing a C compiler when compiling a C compiler seems overly contrived.  There should be no need to remove `CONFIG['dsymutil']`, since the configure script should leave it blank on platforms where it isn't available.  Besides, the CI tests are passing on all platforms as it is now.

> The failures seem coming from codesign, and this file won't need changes after 8a89dd2.

By failures I guess you mean from you running the tests locally with the RUBY_CODESIGN environment variable set.  Thanks for catching that, I didn't realize that would cause the test to fail.